pkg/model/api.go

```diff
@@ -28,15 +28,15 @@ var (
 
 // Api is api gateway concept, control request from browser、Mobile APP、third party people
 type Api struct {
```

Comment:
I think type API is useless now. could we delete it?

Reply:
yes,but maybe fixed in another pr